### PR TITLE
Add accelerate documentation

### PR DESCRIPTION
**Pull request by pimuzzo**
Add documentation for PR https://github.com/docker/distribution/pull/3181

Please don't merge before upstream PR

### CODE REVIEW

**Comment by usha-mandya** [comment]
@pimuzzo Thanks for the PR.  Will wait for the updates to be merged upstream before checking in this update.

File: registry/storage-drivers/s3.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Amazon S3 or S3 compatible services for object storage.
 | `chunksize`  | no | The S3 API requires multipart upload chunks to be at least 5MB. This value should be a number that is larger than 5 * 1024 * 1024.|
 | `rootdirectory`  | no | This is a prefix that is applied to all S3 keys to allow you to segment data in your bucket if necessary. |
 | `storageclass`  | no | The S3 storage class applied to each registry file. The default is `STANDARD`. |
+| `s3accelerate`  | no | Specifies whether the registry should use S3 Transfer Acceleration. You must enable acceleration endpoint on a bucket before using this option. A boolean value. The default is `false`. |
```
Comment (pimuzzo) [inline registry/storage-drivers/s3.md:29]: change the parameter to `accelerate`

Comment (pimuzzo) [inline registry/storage-drivers/s3.md:29]:
```suggestion
| `accelerate`  | no | Specifies whether the registry should use S3 Transfer Acceleration. You must enable acceleration endpoint on a bucket before using this option. A boolean value. The default is `false`. |
```

File: registry/storage-drivers/s3.md [hunk 2]
```diff
@@ -52,6 +53,7 @@ Amazon S3 or S3 compatible services for object storage.
 
 `storageclass`: (optional) The storage class applied to each registry file. Defaults to STANDARD. Valid options are STANDARD and REDUCED_REDUNDANCY.
 
+`s3accelerate`: (optional) Whether you would like to use accelerate endpoint for communication with S3. You must enable acceleration on a bucket before using this option. For details on how to enable the accelerate option, see [Amazon S3 Transfer Acceleration](https://docs.aws.amazon.com/AmazonS3/latest/dev/transfer-acceleration.html).
```
Comment (pimuzzo) [inline registry/storage-drivers/s3.md:56]:
```suggestion
`accelerate`: (optional) Whether you would like to use accelerate endpoint for communication with S3. You must enable acceleration on a bucket before using this option. For details on how to enable the accelerate option, see [Amazon S3 Transfer Acceleration](https://docs.aws.amazon.com/AmazonS3/latest/dev/transfer-acceleration.html).
```

**Comment by pimuzzo** [comment]
@usha-mandya upstream is now merged https://github.com/distribution/distribution/pull/3181

**Comment by usha-mandya** [comment]
Close and reopen PR to trigger checks

**Pull request merged**